index.js

```diff
@@ -8,14 +8,25 @@ module.exports = isLocalIp
 // 172.16.0.0 - 172.31.255.255
 // 192.168.0.0 - 192.168.255.255
 var local = /^(::f{4}:)?(10\.\d{1,3}|127\.\d{1,3}|172\.1[6-9]|172\.2\d|172\.3[0-1]|192\.168)\.\d{1,3}\.\d{1,3}/
+// Matches
```

Comment:
Would it be possible to merge these three new regexes into the existing 'local' regex for consistency?

Reply:
I would prefer the opposite. The local regex is bit too hard to read to be easily maintainable. That's why I put each new ones into a new regex.

Reply:
It's currently just setup as a bunch of 'or's in a regex, however I'm personally fine either way as long is it's consistent. IE, each portion has its own regex or all in one regex joined by ors.

Reply:
I will then split each portion into a regex each tomorrow.